napari_pyclesperanto_assistant/_gui/_category_widget.py

```diff
@@ -17,6 +17,7 @@
 VIEWER_PARAM = "viewer"
 OP_NAME_PARAM = "op_name"
 OP_ID = "op_id"
+OUTPUT_PLACEHOLDER = "output_placeholder"
```

Comment:
since this variable isn't used at all in the building of the widget, just in the generation of the output, probably better to move it to `_Assistant` (or wherever it is actually first assigned)